engine/framework/registry/errors.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package registry
+
+import "github.com/pingcap/tiflow/engine/pkg/rpcerror"
+
+// CreateWorkerRetryableError provides details of ErrCreateWorkerRetryable, which
+// contains original error returned from CreateWorker
+type CreateWorkerRetryableError struct {
+	rpcerror.Error[rpcerror.NotRetryable, rpcerror.Aborted]
```

Comment:
this should be Retryable?

Reply:
`rpcerror.NotRetryable` is used in gRPC client, not the business logic